### PR TITLE
Add Ethereum deploy scripts for exchangeIssuanceZeroEx

### PR DESCRIPTION
**Pull request by cgewecke**
+ Duplicate of #7 for the Ethereum network
+ PR also includes some fixes for the Ethereum folder to support forked network testing - this is the repo's first Ethereum deployment

**Kovan**
+ [0x30BA25b354Cf1A23Ead0Ad84aa81d3E3eD494234][1]

[1]: https://kovan.etherscan.io/address/0x30BA25b354Cf1A23Ead0Ad84aa81d3E3eD494234#code

### CODE REVIEW

File: ethereum/deploy/001_exchangeIssuanceZeroEx.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+} = DEPENDENCY;
+
+const CURRENT_STAGE = getCurrentStage(__filename);
+
+const func: DeployFunction = trackFinishedStage(CURRENT_STAGE, async function (hre: HRE) {
+  const { deploy, deployer } = await prepareDeployment(hre);
+
+  const contractName = CONTRACT_NAMES.EXCHANGE_ISSUANCE_ZEROEX;
+
+  const wethAddress = await findDependency(WETH);
```
Comment (dalberto) [inline ethereum/deploy/001_exchangeIssuanceZeroEx.ts:32]: Minor nit, I didn't note before. We should chain `const` for [brevity]( https://stackoverflow.com/questions/40319891/why-cant-i-declare-multiple-const-in-javascript):

```js
const wethAddress = await findDependency(WETH),
      controllerAddress = await findDependency(CONTROLLER);
```

Comment (cgewecke) [reply ethereum/deploy/001_exchangeIssuanceZeroEx.ts:32]: Whoah! Didn't know this was possible. Awesome.

**Review by dalberto — APPROVED** [review]

**Pull request merged**